src/AnalyticsConfig.ts

```diff
@@ -26,4 +27,10 @@ export interface AnalyticsConfig {
      * This generates a ULID to tie together events in a single browsing session.
      */
     sessionTrackingEnabled?: boolean;
-}
+    /**
+     * Used to force sending the POST request via fetch + keepalive.
```

Comment:
This isn't exactly correct. If the browser supports fetch + keep alive, it'll just use fetch + keepAlive. This flag has no effect on it. 

I would also specify which browsers this would be applicable to.